Common/UnitDefinitions/Power.json

```diff
@@ -112,6 +112,19 @@
           "Abbreviations": [ "J/h" ]
         }
       ]
+    },
+    {
+      "SingularName": "TonOfRefrigeration",
+      "PluralName": "TonsOfRefrigeration",
+      "FromUnitToBaseFunc": "{x} * 3516.8528420667m",
+      "FromBaseToUnitFunc": "{x} / 3516.8528420667m",
```

Comment:
According to wiki, it exactly equals `3.51685 kW`. Do you have a definition to base this on?
https://en.wikipedia.org/wiki/Ton_of_refrigeration

> The modern definition is exactly 12,000 [BtuIT/h](https://en.wikipedia.org/wiki/Power_(physics)) (3.516853 [kW](https://en.wikipedia.org/wiki/Kilowatt))

Reply:
I found that value from this table at [unitconverters.net](https://www.unitconverters.net/power/ton-refrigeration-to-watt.htm)
![image](https://github.com/user-attachments/assets/08208ba8-d279-41d4-84a4-1f7351abe61d)

Since it started with the same numbers I assumed it was a more precise version of the one listed on wikipedia and the number was irrational.

Digging further from the [source listed from wikipedia](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication811e2008.pdf) I can see it is indeed listed the same as the wikipedia so I will go ahead and change it to that.

![image](https://github.com/user-attachments/assets/8ea57d0c-7a2d-49b5-9a4e-0d213101df94)

Reply:
I just pushed the adjustment. Let me know if there is anything else. Thanks!

Reply:
Thanks, yes I have found that online converters are not always very reliable on precise values. They probably do some calculations that introduce floating point errors.

Wiki is my goto source, since you can typically find the source they refer to.